starport/pkg/protobufjs/protobufjs.go

```diff
@@ -66,6 +64,10 @@ func Generate(ctx context.Context, outDir, outName, protoPath string, includePat
 
 	// add proto dependency paths to that.
 	for _, includePath := range includePaths {
+		if _, err := os.Stat(includePath); os.IsNotExist(err) {
```

Comment:
We should handle `err != nil && !os.IsNotExist(err)`

Reply:
removed by 0dbe9de.